Sources/SwifterSwift/CryptoKit/DigestExtensions.swift

```diff
@@ -0,0 +1,17 @@
+// DigestExtensions.swift - Copyright 2022 SwifterSwift
+
+#if canImport(CryptoKit)
+import CryptoKit
+
+@available(iOS 13.0, macOS 10.15, watchOS 6.0, tvOS 13.0, *)
+public extension Digest {
+
+    // MARK: - Properties
+
+    /// SwifterSwift: Hexadecimal value string (read-only)
```

Comment:
Can we have a Complexity doc as well, to make clear for people that we have a linear `O(n)` at least in this computed property?

Reply:
Added with 42e2959a4b2b11b05b26f18ad12b10da4a734faa